scripts/retdec.py

```diff
@@ -1,4 +1,18 @@
-class RetDecCommand(GenericCommand):
+import getopt
+import os
+import re
+import sys
+import tempfile
+
+import gdb
+
+from gef import (GEF_TEMP_DIR, PYTHON_MAJOR, current_arch, err, get_arch,
```

Comment:
Why not just hcange the script to use `gef.<name>`?

Reply:
Fewer lines changed, more flexibility when refactoring, shorter lines that are easier to read and faster runtime (avoid the dots).